packages/react/src/filter-sidebar/FilterSidebar.tsx

```diff
@@ -0,0 +1,30 @@
+import { PropsWithChildren } from 'react';
+import { FormStack } from '@ag.ds-next/react/form-stack';
+import { CollapsingSideBar } from '../_collapsing-side-bar';
+
+export type FilterSidebarProps = PropsWithChildren<{
+	/** The aria-label for the FilterSidebar. */
+	'aria-label'?: string;
+	numberOfActiveFilters?: number;
```

Comment:
This is a bit of a nit: but I think `activeFiltersCount` would be a little more clearer. Could we also add a prop comment here?